content/analyst/inbox-and-design-tracker/reputation-advisor-scheduled-report.md

```diff
@@ -0,0 +1,44 @@
+---
+lastUpdated: "02/23/2023"
+title: Reputation Advisor Scheduled Report
+description: Take your reputation monitoring to the next level by configuring a weekly audit straight to your inbox.
+---
+
+## Let eDataSource run over 100+ checks on your deliverability each day or week and alert you when things change.
```

Comment:
Do we want to change this heading (`eDataSource`)?

Reply:
Not gonna change too much from the original articles